wfl/autoparallelize/pool.py

```diff
@@ -102,7 +102,7 @@ def do_in_pool(npool=None, chunksize=1, iterable=None, configset_out=None, op=No
 
     did_no_work = True
 
-    if isinstance(iterable, ConfigSet_in):
+    if isinstance(iterable, ConfigSet):
         items_inputs_generator = grouper(chunksize, ((item, iterable.get_current_input_file()) for item in iterable))
     else:
         items_inputs_generator = grouper(chunksize, ((item, None) for item in iterable))
```

Comment:
Line 112 often translates to 
```
 Running <function evaluate_op at 0x145549c97160> with npool=16, chunksize=1
```
which is not perfectly helpful, because most of the `op`'s names aren't very helpful. Could this also report the module of the function? Or alternatively, should we give better names to all of `*autopara_wrappable' functions?

Reply:
I agree that it's not helpful.  I wasn't thinking that they needed to have a sensible name, but because of this sort of thing they do. Maybe do that as part of the `autoparallelize` manual wrap to `iloop` that's in issue #48 ?

Reply:
Sounds good, I've just made a note there.